server/buttonAlerts.js

```diff
@@ -13,12 +13,12 @@ function setup(braveAlerterObj) {
   braveAlerter = braveAlerterObj
 }
 
-async function handleValidRequest(button, numButtonPresses) {
+async function handleValidRequest(button, numberOfAlerts) {
```

Comment:
This one might actually be worth leaving as `numButtonPresses` or taking out entirely. 

This function is called from `rak.js:handleButtonPress` each time it's called. Nowadays, it is always given the value of one. In the past, when we had Flic Buttons, they had the ability to send us a double press and we would count that as two presses, but it would still have generated just a single alert.

Reply:
Oh, interesting. Removing it is probably the best action here then; I'll do that!